ste/mgr-JobPartTransferMgr.go

```diff
@@ -642,7 +642,17 @@ func (jptm *jobPartTransferMgr) failActiveTransfer(typ transferErrorCode, descri
 		if status == http.StatusForbidden {
 			// quit right away, since without proper authentication no work can be done
 			// display a clear message
-			common.GetLifecycleMgr().Error(fmt.Sprintf("Authentication failed, it is either not correct, or expired, or does not have the correct permission %s", err.Error()))
+			common.GetLifecycleMgr().Info(fmt.Sprintf("Authentication failed, it is either not correct, or expired, or does not have the correct permission %s", err.Error()))
+			// and use the normal cancelling mechanism so that we can exit in a clean and controlled way
+			jobId := jptm.jobPartMgr.Plan().JobID
+			CancelPauseJobOrder(jobId, common.EJobStatus.Cancelling())
+			// TODO: this results in the final job output line being: Final Job Status: Cancelled
+			//     That's not ideal, because it would be better if it said Final Job Status: Failed
```

Comment:
Reporting it as Cancelled seems fine, since that's exactly what we did. Cancellation doesn't have to be initiated by user